src/onboarding/sendEmail.js

```diff
@@ -30,9 +65,9 @@ function sendEmail() {
         subject, 
         "", 
         {
-          "from":"services@coderbunker.com", 
-          "name": "Coderbunker Services", 
-          "cc":"bizdev@coderbunker.com",
+          from: "services@coderbunker.com", 
+          name: "Coderbunker Services", 
+          cc: "bizdev@coderbunker.com",
```

Comment:
conditional testing flag here?

Reply:
You can file an issue. I'm going to do that in the future version